sys/usb/usbus/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_USBUS
+    bool "Configure USB Unified Stack (USBUS)"
+    depends on MODULE_USBUS
+    help
+        Configure the USBUS module via Kconfig.
+
+if KCONFIG_MODULE_USBUS
+
+config USBUS_AUTO_ATTACH
+    bool "Auto attach"
+    default y
+    help
+        When set, the USBUS thread will automatically enable the USB pull-up
+        resistor after initializing the thread. This will signal to the host
+        that the USB peripheral is ready for use.
+
+config USBUS_EP0_SIZE
+    int "Endpoint 0 buffer size"
+    default 64
```

Comment:
As per USB spec, the only valid values for this buffer size are 8, 16, 32 and 64. Should this option be a choice type with the valid buffer options?

Reply:
Yes, sure. I was not aware of that constraint. I modeled this as a choice now.